clang/docs/StandardCPlusPlusModules.rst

```diff
@@ -652,6 +652,134 @@ in the future. The expected roadmap for Reduced BMIs as of Clang 19.x is:
    comes, the term BMI will refer to the Reduced BMI and the Full BMI will only
    be meaningful to build systems which elect to support two-phase compilation.
 
+Experimental No Transitive Change
```

Comment:
Is it actually experimental if it's the default and there is no way to disable it?
I suggest saying "initial", would it better to capture what we intend to do (I suspect we want to keep improving this and want feedback where it falls short)?

Reply:
I'd like to find a better term than "no transitive change" -- this doesn't work well as a noun phrase in English, so we say awkward things like "...the no transitive change feature can be more powerful if ..." I think what bothers me about is the "no" and "change" part -- "change" is a verb, so it comes out sounding verb-like and we end up tacking on "feature" to try to make it a noun again.

That said, I don't have a significantly better phrase currently. :-)

Reply:
* "no-cascade" feature?
* "isolated BMI" (not sure if this is a great idea given how there's already full/reduced BMI)
* "BMI isolation"
* + variations with synonyms for isolated/secluded/confined/...

Reply:
A slightly cleaner formulation might be "non-cascading X" or perhaps "non-percolating Y".

Reply:
> On "experimental": thanks for clarifying that you meant this as "experimental in the build system". I am afraid that the current wording actually does the opposite and tells that this the compiler support is "experimental" (aka "unstable", aka "don't touch this until it stabilizes").

Yeah, and I like to call it as `experimental with the build system`. Since although the users can only get it from build systems, the core abilities are provided by the compiler. Also when bug reports comes, I believe most of them need to solve in the compiler side. But this doesn't matter though : )